omnigibson/controllers/joint_controller.py

```diff
@@ -132,6 +128,23 @@ def is_grasping(self):
         # No good heuristic to determine grasping, so return UNKNOWN
         return IsGraspingState.UNKNOWN
 
+    def compute_no_op_command(self, control_dict):
+        # Compute based on mode
+        if self.use_delta_commands:
+            if self._motor_type == "position":
+                # Zero values
+                cmd = np.zeros(self.command_dim)
+            elif self._motor_type == "velocity" or self._motor_type == "effort":
+                # Run negative delta
+                cmd = -control_dict["joint_{}".format(self._motor_type)][self.dof_idx]
+            else:
+                raise ValueError(f"Got invalid motor type: {self._motor_type}!")
+        else:
+            # Directly use info from control dict
+            cmd = control_dict["joint_{}".format(self._motor_type)][self.dof_idx]
```

Comment:
hmm I am confused, I thought no op wants the robot to stay still. 
For absolute control, shouldn't we do the following?
- position: cmd = current joint position
- velocity/torque: 0

Reply:
oops! you're right, great catch. fixed!